lib/OnyxConnectionManager.ts

```diff
@@ -73,12 +73,18 @@ class OnyxConnectionManager {
      */
     private lastCallbackID: number;
 
+    /**
+     * Stores the last generated session ID for the connection manager.
```

Comment:
Maybe we can add some explanation about what this `sessionID` is used for and why we have it at a ll?